core/flyout_base.js

```diff
@@ -486,6 +486,7 @@ Blockly.Flyout.prototype.show = function(xmlList) {
       case 'BUTTON':
         var curButton = new Blockly.FlyoutButton(this.workspace_,
```

Comment:
It might be a little easier to read if you explicitly set isLabel like this.
case 'LABEL':	
￼case 'BUTTON':
        // Labels behave the same as buttons, but are styled differently.	
        var isLabel = xml.tagName.toUpperCase() == 'LABEL';

Reply:
Changed!